.github/workflows/kafka-topics-dev.yaml

```diff
@@ -0,0 +1,23 @@
+name: Deploy kafka topics to dev
+on:
+  push:
+    paths:
+      - "kafka-topics/dev/*.yaml"
+
+jobs:
+  deploy-dev:
+    name: Deploy kafka topics to dev
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
Kan benytte `actions/checkout@v3` i stedet (bruker nyere node-versjon, samt at den kun sjekker ut siste commit i stedet for hele repo, om jeg husker riktig..)

Reply:
Godt poeng, jeg fikser.